### PR TITLE
Release 1.6.0

### PR DESCRIPTION
**Pull request by Rashmil-1999**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
```
Comment (longshuicy) [inline CHANGELOG.md:8]: On release branch this change to the release [1.6.0] with a date (next Tuesday maybe?)

**Review by longshuicy — CHANGES_REQUESTED** [review]
You also need to do a project-wide search of the version 1.5.0 and replace them with 1.6.0

File: CHANGELOG.md (hunk 1 above)
Comment (Rashmil-1999) [reply CHANGELOG.md:8]: Okay, I will pull the release branch and do that. It slipped out of my mind, sorry.

**Comment by Rashmil-1999** [comment]
@longshuicy I updated the version numbers. Could you jut check them to make sure I have done it at all possible and required places?
Also, is there anything else that needs to be done with the release branch?

**Review by longshuicy — APPROVED** [review]
Looks good.

**Review by navarroc — APPROVED** [review]

**Pull request merged**